src/api/hooks/useParkingSpots.js

```diff
@@ -0,0 +1,20 @@
+import { fetchQuery, Queries } from '@/api/base';
+import { useQuery } from '@tanstack/react-query';
+
+const API_BACKEND_URL = import.meta.env.VITE_API_BACKEND_URL;
+
+export const useParkingSpots = ({ queryClient }) => {
+  const query = useQuery({
+    queryKey: [Queries.ParkingSpots],
+    queryFn: fetchQuery({
+      url: `${API_BACKEND_URL}/parking-lots`, // @TODO: Update the URL with the /api proxied path
```

Comment:
@ztevenx100 @CamiloLeonP I've added this @TODO because I wasn't able to link here properly the proxy I've just created using React Query. It works perfectly with native fetch but it behaves weird with this new implementation.

Reply:
Will be addressed in https://github.com/parkiusource/parking-radar-frontend/issues/29